### ISSUE TITLE
Furaffinity won't download scraps, only galleries

### ISSUE THREAD

**Issue by Wiiplay123** [4pr0n/ripme#249]
I don't have much time to work on my own fork right now for this issue, but it seems very simple to fix.

**Comment by Wiiplay123** [comment]
Also, I'm getting Index: 0, Size 0 when ripping FurAffinity galleries.

**Comment by Wiiplay123** [comment]
Definitely would be nice to have Ripme remade in C++ or something.

**Comment by Wiiplay123** [comment]
Right, I'm actually getting Error: null on latest version. According to Eclipse, threads are being rejected after about 60 are queued on the furaffinity ripper.

**Comment by metaprime** [comment]
Please provide some example URLs and expected vs. actual behavior.

**Comment by rautamiekka** [comment]
I wonder if @Wiiplay123 used an older version of RipMe and/or Java ? As (RipMe 1.2.13, Oracle Java SE 8u112 x64, Window$ 7 Ultimate SP1 x64) when I'm trying any user's scraps section I get URL syntax violation, it doesn't give that for an user's entire gallery, but it fails to DL cuz of supposedly no images found at address although unlogged in visit reveals at least some of it.

**Comment by metaprime** [comment]
@rautamiekka Thanks for the investigation! Since some of these issues are so old, I'm sure that the picture has changed a lot.
